oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -673,7 +679,12 @@ sshmem_mkey_t *mca_spml_ucx_register(void* addr,
         return NULL;
     }
 
-    segno   = memheap_find_segnum(addr);
+    segno   = memheap_find_segnum(addr, my_pe);
+    if (MEMHEAP_SEG_INVALID == segno) {
+        SPML_UCX_ERROR("mca_spml_ucx_register failed because of invalid \
+        segment number: %d\n", segno);
```

Comment:
please don't use ```\``` as long line delimiter. C allows to concatenate strings like
```
SPML_UCX_ERROR("mca_spml_ucx_register failed because of invalid "
        "segment number: %d\n", segno);```